projects/counterstake/index.js

```diff
@@ -72,78 +74,43 @@ async function totalObyteTvl(timestamp) {
-
-    const chain = toUpperCaseFirstLetter(api.chain === 'bsc' ? "BSC" : api.chain);
+    const bridges = await getConfig('counterstake/bridges', 'https://counterstake.org/api/bridges').then((data) => data.data);
+    const pooledAssistants = await getConfig('counterstake/poolStakes', 'https://counterstake.org/api/pooled_assistants').then((data) => data.data);
 
-    const erc20Contracts = [];
-    const nativeContracts = [];
     const bridgeAasByChain = [];
+    const tokensAndOwners = []
 
     bridges.forEach(({ home_network, foreign_network, export_aa, import_aa, stake_asset, home_asset }) => {
-        if (home_network === chain) { // export
-            if (home_asset !== ZERO_ADDRESS) {
-                erc20Contracts.push({ contract: export_aa, tokenContract: home_asset });
-            } else {
-                nativeContracts.push({ contract: export_aa, tokenContract: home_asset });
-            }
+        if (home_network.toLowerCase() === api.chain) { // export
+            tokensAndOwners.push([home_asset, export_aa])
             bridgeAasByChain.push(export_aa);
-        } else if (foreign_network === chain) { // import
-            if (stake_asset !== ZERO_ADDRESS) {
-                erc20Contracts.push({ contract: import_aa, tokenContract: stake_asset });
-            } else {
-                nativeContracts.push({ contract: import_aa, tokenContract: stake_asset });
-            }
+        } else if (foreign_network.toLowerCase() === api.chain) { // import
+            tokensAndOwners.push([stake_asset, import_aa])
             bridgeAasByChain.push(import_aa);
         }
     });
 
-    pooledAssistants.filter(({ network }) => network === chain).forEach(({ assistant_aa, side, bridge_id }) => {
+    pooledAssistants.filter(({ network }) => network.toLowerCase() === api.chain).forEach(({ assistant_aa, side, bridge_id }) => {
         const bridge = bridges.find((bridge) => bridge.bridge_id === bridge_id);
 
         if (bridge) {
             if (side === 'import') {
                 // stake asset
-                if (bridge.stake_asset !== ZERO_ADDRESS) {
-                    erc20Contracts.push({ contract: assistant_aa, tokenContract: bridge.stake_asset });
-                } else {
-                    nativeContracts.push({ contract: assistant_aa, tokenContract: bridge.stake_asset });
-                }
-
+                tokensAndOwners.push([bridge.stake_asset, assistant_aa])
                 // imported asset
-                if (bridge.foreign_asset !== ZERO_ADDRESS) {
-                    erc20Contracts.push({ contract: assistant_aa, tokenContract: bridge.foreign_asset });
-                } else {
-                    nativeContracts.push({ contract: assistant_aa, tokenContract: bridge.foreign_asset });
-                }
+                // tokensAndOwners.push([bridge.foreign_asset, assistant_aa])
```

Comment:
@g1nt0ki Hello, thanks for the refactoring. Why is it commented out?

Reply:
we count the assets only on the source side, else it would be double counting.